File: fs_project/fs_project/ext_glue.fs
(*******************************
This module is very important. It
defines the interface between the
python Omniverse extentsion framework
and the Fable F# code.

To create a new extension, modify
the defined functions to run your
F# code

********************************)

module ext_glue

let ext_on_startup extid =
    printfn("in startup")

let ext_on_shutdown() =
    printfn("in startup")

File: fs_project/fs_project/ext_glue.py
from typing import Any
from .fable_modules.fable_library.string_ import (to_console, printf)

def ext_on_startup(extid: Any | None=None) -> None:
    to_console(printf("in startup"))


def ext_on_shutdown(__unit: None=None) -> None:
    to_console(printf("in startup"))


__all__ = ["ext_on_startup", "ext_on_shutdown"]

